src/main/fc/fc_core.c

```diff
@@ -141,6 +141,7 @@ static timeUs_t runawayTakeoffTriggerUs = 0;
 static bool runawayTakeoffTemporarilyDisabled = false;
 #endif
 
+static bool graveyardModeDisabled = true;
```

Comment:
I'd turn this around and call it `graveyardModeEnabled`.

Reply:
I thought about that variable name quite a lot actually. `graveyardModeEnabled` sounds like it is currently is enabled, which isn’t necessarily the case. It tracks if it can be enabled because the mode was off at least once (to prevent forgetting to flip the switch back at least once). I’ll try to find a better name.

Reply:
How is `graveyardModeAllowed`?